### ISSUE TITLE
docker模式安装报错

### ISSUE THREAD

**Issue by foolbear666** [kroese/virtual-dsm#141]
您好，我运行命令  docker run -it --rm -p 5000:5000 --device=/dev/kvm --cap-add NET_ADMIN --stop-timeout 60 kroese/virtual-dsm:latest之后 提示以下信息：
Starting Virtual DSM for Docker v3.70...
Install: Downloading extractor...
ERROR: Failed to download extractor, reason: 6

我的操作系统是centos 7.7 1908, 不明白这个问题是什么原因导致

**Comment by offzen** [comment]
I got an error 'reason: 28'
But I can wget that .pat file directly.

**Comment by offzen** [comment]
WHEN I SET 'COUNTRY: "US"
-------------

bash /run/progress.sh /storage/tmp/rd.gz Downloading installer ([P])...
--
root | 27839 | 27775 | 0 | 22:28 | ? | 00:00:00 | curl -r 65627648-71021835 -sfk -S -o /storage/tmp/rd.gz https://global.synologydownload.com/download/DSM/release/7.0.1/42218/DSM_VirtualDSM_42218.pat
root | 28165 | 27838 | 0 | 22:30 | ? | 00:00:00 | sleep 1

-----AFTER SET SET 'COUNTRY: "CN"--------is it strange that it always try to download the older version 7.0.1 instead 7.2.1?------
❯ Starting Virtual DSM for Docker v7.01...
❯ For support visit https://github.com/vdsm/virtual-dsm
❯ Install: Downloading installer...
curl: (22) The requested URL returned error: 416
❯ ERROR: Failed to download https://cndl.synology.cn/download/DSM/release/7.0.1/42218/DSM_VirtualDSM_42218.pat, reason: 22

**Comment by offzen** [comment]
and I tried to add an URL( .pat reachable), but seems it would be ignored. The error 'reason:28' came up.

> environment:
>           URL: "http://192.168.3.2:81/DSM_VirtualDSM_42218.pat"

**Comment by offzen** [comment]
You're right about the older version of the image. After update image, I'm trying to solve the network through a (export http) proxy server.
Still got error "reason 28", hoping it's a problem of the proxy.
Thx!

**Comment by offzen** [comment]
Cool! The new feature ENV HTTP works so smoothly! SUPER HELPFUL! 
Thank you!
